ViewGenerator/build/ViewGenerator.targets

```diff
@@ -43,13 +43,13 @@
       <Exec Condition="'$(OS)' != 'Windows_NT'" Command="ln -s &quot;$(ProjectDir)$(PluginsRelativePath)node_modules&quot; &quot;$(ProjectDir)node_modules/webview.plugins&quot;" />
   </Target>
 
-  <Target Name="VG_UnlinkNodeModules">     
+  <Target Name="VG_UnlinkNodeModules">
     <RemoveDir Condition="Exists('$(ProjectDir)node_modules\webview.plugins')" Directories="$(ProjectDir)node_modules\webview.plugins" />
   </Target>
-    
+
   <Target Name="VG_CleanEntryFiles" >
     <ItemGroup>
-      <EntryFilesToDelete Include="**\*.entry" />
+      <EntryFilesToDelete Include="**\*.entry" Exclude="node_modules\**\*.entry" />
```

Comment:
Which particular package had files with this extension?

Reply:
Using workspaces in pnpm you can replace the symbolic link to webview.plugins with a package. But internally a symbolic is created so files were being deleted.

![image](https://user-images.githubusercontent.com/74782571/236776017-45bd9477-a726-491f-93ad-69282500e552.png)